### PR TITLE
Ogre Ethnicity

### PR DESCRIPTION
**Pull request by ercarp**
## Changelog:
- Created a custom ethnicity for ogres using `orcish_rework` ethnicity as a template.
- Made a clothing set for ogre cultures using Hindu and Buddhist religious clothes.

# How to test:
There are two ways you can test this:
1. Open up Ruler Designer and generate male and female ogres to see what they look like. The ethnicity is currently called simply `ogre_ethnicity`.
2. Alternatively, load up into a game and look for ogres in-game by using the character search panel.

Give feedback and suggestions on what, if anything, could or should be improved or changed.

## Known issues

- Due to the chonkiness of ogres, some animations cause their hands to clip through their own bodies. This is a problem with Vanilla CK3 and will not be fixed.

### CODE REVIEW

**Review by TheGroxEmpire — COMMENTED** [review]
Some of the top of their heads are too narrow, they look to be squished and weird to me:
<details>

image
image
</details>

This one is I think a good proportion:
<details>

image

</details>

**Comment by zumbak04** [comment]
> Some of the top of their heads are too narrow, they look to be squished and weird to me:
 I would agree, on the first screenshot he's too small brain.

**Comment by ercarp** [comment]
Changed head top width in 0a1e6e1ac3ef63ce5f475aedc6ca73f6ff32db1e.

**Comment by TheDrPibb1999** [comment]
Would it be possible to add the horn/shaved horn to the top of their heads such as bellow?
ogre

ogre

or an even more daunting suggestion is the infamous 2 headed ogre, a rarer occurrence or maybe just for important ogres such as cho'gall, but I can understand if that is way down the road

**Comment by TheDrPibb1999** [comment]
accidentally put the same photo twice, meant to put 
ogre 2

**Review by TheDrPibb1999 — APPROVED** [review]
With my horns question being answered by the simple text above, and my only inquiry being the double-headed ogre. I had no issues with generating a multitude of ogres with nice-sized brains, all ogrey. Great model!

**Comment by ercarp** [comment]
> or an even more daunting suggestion is the infamous 2 headed ogre, a rarer occurrence or maybe just for important ogres such as cho'gall, but I can understand if that is way down the road

One-eyed ogres and two-headed ones are indeed _waaaaaaaaaay_ down the line. Both are probably doable, but getting them to work and look right is a very time-consuming process and therefore not a priority at the moment.

**Review by TheGroxEmpire — COMMENTED** [review]
- Top of the head looks good now.
- The only canon image of female Ogre depicts them as having hairs. I think we should to. (https://wow.gamepedia.com/Karrga)

File: common/culture/cultures/ogre.txt [hunk 1]
```diff
@@ -2,15 +2,12 @@
 	graphical_cultures = {
```
Comment (zumbak04) [inline common/culture/cultures/ogre.txt:2]: Reminder: add hairs and beards.

Comment (zumbak04) [inline common/culture/cultures/ogre.txt:2]: Do you plan to add Mok'nathal here or on another branch?

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/culture/cultures/ogre.txt (hunk 1 above)
Comment (ercarp) [reply common/culture/cultures/ogre.txt:2]: > Do you plan to add Mok'nathal here or on another branch?

Haven't thought about it yet. I suppose I could just add them to this one.

Comment (ercarp) [reply common/culture/cultures/ogre.txt:2]: > 
> 
> Reminder: add hairs and beards.

On it!

Comment (ercarp) [reply common/culture/cultures/ogre.txt:2]: Added in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/188.

**Review by TheGroxEmpire — COMMENTED** [review]
Imo the horn could use some tweaks. Since they are really big, the low poly and low res looks of it kinda puts it off compared to how polished the rest are. It's still not that bad for a basic implementation. We could maybe improve it later when we get a 3D artist if you reckon to.

**Review by DatFrenchBanana — APPROVED** [review]
Works very well for all the ogres in the game. They look very good in my games.

**Pull request merged**